index.html

```diff
@@ -18,7 +18,7 @@
                     <li><a href="#">Portfolio</a></li>
```

Comment:
- [ ] One of the requirements of the project, is to prepare a report listing all points that you have checked, and the result of your evaluation. 
- [ ] Please add your report to the description of your pull request.
## Example
![image](https://user-images.githubusercontent.com/28109626/156248698-c263da7e-1526-4805-a913-74fbf34d156a.png)